scripts/update-all.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/sh
+
+NEW_VERSION=$(npm show prisma@$1 version)
+echo "$NEW_VERSION" > .github/prisma-version.txt
+
+pnpm -rc --parallel exec "$(pwd)/scripts/update-version.sh $NEW_VERSION"
+pnpm -rc --parallel exec "$(pwd)/scripts/update-locks.sh $NEW_VERSION"
```

Comment:
This runs the script on all the projects/package.json it can find?

Reply:
Yes

Reply:
(can this be marked as resolved?)